code/_onclick/click.dm

```diff
@@ -77,8 +77,10 @@
 
 	if(SEND_SIGNAL(src, COMSIG_MOB_CLICKON, A, params) & COMSIG_MOB_CANCEL_CLICKON)
 		return
-
 	var/list/modifiers = params2list(params)
+	if(LAZYACCESS(modifiers, RIGHT_CLICK))
+		if(RightClickOn(A))
```

Comment:
I'm very wary of this, and probably need to do some testing of my own.

Are you CONFIDENT this preserves the behavior there is now, where if I right-click something it'll act like a left-click if nothing hooks to this? As in--right clicking items still picks them up, right-clicking a machine will still use it, etc.

Reply:
Good catch, you're not wrong. It did end up removing RClick's passive functionality of being LClick.